docs/managing-documents/reading.md

```diff
@@ -114,6 +114,12 @@ print(docs)
 Please see [the application developer's guide](https://docs.marklogic.com/guide/rest-dev/documents#id_80116)
 for more information on reading documents.
 
+## Obtaining the raw HTTP response
```

Comment:
I learned recently that "raw" means something in `requests` - it's a property of the `response` object - e.g. https://stackoverflow.com/questions/16923898/how-to-get-the-raw-content-of-a-response-in-requests-with-python . So we should avoid saying "raw". 

I think the terminology in your test is good - i.e. "Returning the original response". So just swap "obtain" for "return" and "raw" for "original".